New version: MakieThemes v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: MakieThemes
- Repository: https://github.com/MakieOrg/MakieThemes.jl
- Created by: @asinghvi17
- Version: v0.1.1
- Commit: 2ad534a0e98fa9ce1563c45eb4ffa6f1dfe13e90
- Reviewed by: @asinghvi17
- Reference: https://github.com/MakieOrg/MakieThemes.jl/commit/2ad534a0e98fa9ce1563c45eb4ffa6f1dfe13e90#commitcomment-138301169
- Description: Themes for Makie
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd625f859159da353f369f2e0bb3a551a346649a97f285a2af0d4bebde210cec29530503dafedee2cc9c628d08fb36050742fae5cacac2f9ad5ee32f3fb845ced7f4d984472921812c57255f5cb20be7f4085721a5473058f52c486453b83ee417f4eec2372e55288e4edc76e3c86ec8bf5bc9e1b8c09836cb0a33081d9351e38f9eb7f33a19254d3ff9d8642e3d1c0d8308fcf444bdf64366d38d107cd595353f5b9f6f9e9b86f1fa76c455178c559a3a2 -->

asinghvi17 commented:
Need to block this - someone reported an issue in the theme that has to be fixed!  Sorry for the noise.

JuliaRegistrator committed:
New version: MakieThemes v0.1.1

UUID: e296ed71-da82-5faf-88ab-0034a9761098
Repo: https://github.com/MakieOrg/MakieThemes.jl.git
Tree: 223afe626c0cc901e4f6337973ad96119f889833

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
